src/constants/AboutUs.js

```diff
@@ -7,6 +7,8 @@ const ABOUT_US = {
   FIND_EVENTS: "Find events",
   GET_CONNECTED: "Get connected",
   HAVE_FUN_TOGETHER: "Have fun together",
+  LOREM_IPSUM:
```

Comment:
If you replace it with real text, LOREM_IPSUM variable name will confuse ppl. Rename to TEXT_INFO

Reply:
Done